### ISSUE TITLE
Swiping multiple Dismissables leads to inconsistent behavior

### ISSUE THREAD

**Issue by mehmetf** [flutter/flutter#49488]
In a `ListView` with Dismissable widgets, swiping 2-3 tiles at the same time leads to inconsistent behavior. Sometimes all items are dismissed, sometimes a single item is dismissed and others pop back into the list.

This is observable within the Gallery app.

**Comment by markusaksli-nc** [comment]
This seems fixed on the latest master `1.22.0-10.0.pre.356`
<details>
 <summary>flutter doctor -v</summary>

```
[√] Flutter (Channel master, 1.22.0-10.0.pre.356, on Microsoft Windows [Version 10.0.19041.508], locale en-US)
    • Flutter version 1.22.0-10.0.pre.356 at C:\Development\flutter_master
    • Framework revision c412fd9967 (2 hours ago), 2020-09-24 22:47:03 -0700
    • Engine revision 435db234e4
    • Dart version 2.11.0 (build 2.11.0-161.0.dev)

[√] Android toolchain - develop for Android devices (Android SDK version 30.0.1)
    • Android SDK at C:\Users\marku\AppData\Local\Android\sdk
    • Platform android-30, build-tools 30.0.1
    • Java binary at: C:\Users\marku\AppData\Local\JetBrains\Toolbox\apps\AndroidStudio\ch-0\193.6626763\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)
    • All Android licenses accepted.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files (x86)\Google\Chrome\Application\chrome.exe

[√] Visual Studio - develop for Windows (Visual Studio Community 2019 16.6.5)
    • Visual Studio at C:\Program Files (x86)\Microsoft Visual Studio\2019\Community
    • Visual Studio Community 2019 version 16.6.30320.27
    • Windows 10 SDK version 10.0.18362.0

[√] Android Studio (version 4.0)
    • Android Studio at C:\Users\marku\AppData\Local\JetBrains\Toolbox\apps\AndroidStudio\ch-0\193.6626763
    • Flutter plugin version 47.1.2
    • Dart plugin version 193.7361
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)

[√] Connected device (6 available)
    • SM G950F (mobile)           • ce12171c51cc001c03 • android-arm64  • Android 9 (API 28)
    • sdk gphone x86 arm (mobile) • emulator-5554      • android-x86    • Android 11 (API 30) (emulator)
    • Windows (desktop)           • windows            • windows-x64    • Microsoft Windows [Version 10.0.19041.508]
    • Web Server (web)            • web-server         • web-javascript • Flutter Tools
    • Chrome (web)                • chrome             • web-javascript • Google Chrome 85.0.4183.121
    • Edge (web)                  • edge               • web-javascript • Microsoft Edge 85.0.564.51

• No issues found!
```
</details>
<details>
 <summary>Minimal code sample</summary>

```dart
import 'package:flutter/material.dart';

void main() => runApp(MyApp());

class MyApp extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: 'Sample',
      home: Scaffold(
          appBar: AppBar(
            title: Text('Sample'),
          ),
          body: ListView.builder(
            itemBuilder: (BuildContext context, int index) {
              return Dismissible(
                key: Key('dismissible-$index'),
                child: Card(
                  child: Container(
                    child: Text('Dismissible $index'),
                    height: 50,
                  ),
                ),
              );
            },
            itemCount: 15,
          )),
    );
  }
}
```
</details>

<img src="https://user-images.githubusercontent.com/68219924/94258229-34a80e80-ff35-11ea-98c6-a62284eef72a.gif" height="600" />

Should we close this @mehmetf?

**Comment by TahaTesser** [comment]
Without additional information, we are unfortunately not sure how to resolve this issue. 
We are therefore reluctantly going to close this bug for now. 
Please don't hesitate to comment on the bug if you have any more information for us; we will reopen it right away!  
Thanks for your contribution.
